src/SFML/Graphics/ImageLoader.cpp

```diff
@@ -61,6 +61,15 @@ namespace
         sf::InputStream* stream = static_cast<sf::InputStream*>(user);
         return stream->tell() >= stream->getSize();
     }
+
+    // stb_image callback for constructing a buffer
+    // This assumes 'context' is an std::vector<sf::Uint8>*
+    void bufferFromCallback(void* context, void* data, int size)
+    {
+        sf::Uint8* ptr = &static_cast<sf::Uint8*>(data)[0];
+        sf::Uint8* end = ptr + size;
```

Comment:
Minor comment: I think this is overly verbose. I'm personally fine with a direct `...(ptr, ptr + size)`.

Reply:
New implementation.